helm-charts/mock-classifier/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This is a smart way to test - great idea! 💡

Reply:
I'd love to take credit, but this was auto generated 😆 I have added this issue https://github.com/buerokratt/Infrastructure/issues/24 to keep track and build the tests out